src/main/webapp/app/admin/cleanup-service/cleanup-service.component.html

```diff
@@ -0,0 +1,76 @@
+<div>
+    <h2>
+        <span id="cleanup-page-heading" jhiTranslate="cleanupService.title">Cleanup Service Operations</span>
+    </h2>
+    <div class="table-responsive">
+        <table id="cleanupOperations" class="table table-striped" aria-describedby="cleanup-page-heading">
+            <thead>
+                <tr>
+                    <th scope="col" jhiTranslate="cleanupService.table.operation">Operation</th>
+                    <th scope="col" jhiTranslate="cleanupService.table.deleteFrom">Delete From</th>
+                    <th scope="col" jhiTranslate="cleanupService.table.deleteTo">Delete To</th>
+                    <th scope="col" jhiTranslate="cleanupService.table.lastExecuted">Last Executed</th>
+                    <th scope="col"></th>
+                </tr>
+            </thead>
+            <tbody>
+                @for (operation of cleanupOperations; track operation) {
+                    <tr>
+                        <td>
+                            {{ 'cleanupService.operation.' + operation.name | artemisTranslate }}
```

Comment:
Could we use a `span` here? Something like

```suggestion
                            <span [jhiTranslate]="'cleanupService.operation.' + operation.name"></span>
```

Reply:
done